nhr-loader/src/main/java/nl/b3p/brmo/nhr/model/Persoon.java

```diff
@@ -0,0 +1,240 @@
+/*
+ * Copyright (C) 2022 B3Partners B.V.
+ *
+ * SPDX-License-Identifier: MIT
+ *
+ */
+
+package nl.b3p.brmo.nhr.model;
+
+import nl.kvk.schemas.schemas.hrip.catalogus._2015._02.PersoonType;
+import nl.kvk.schemas.schemas.hrip.catalogus._2015._02.NaamPersoonType;
+import nl.kvk.schemas.schemas.hrip.catalogus._2015._02.NatuurlijkPersoonType;
+import nl.kvk.schemas.schemas.hrip.catalogus._2015._02.NietNatuurlijkPersoonType;
+import nl.kvk.schemas.schemas.hrip.catalogus._2015._02.RechtspersoonType;
+import nl.kvk.schemas.schemas.hrip.catalogus._2015._02.SamenwerkingsverbandType;
+import nl.kvk.schemas.schemas.hrip.catalogus._2015._02.RechtspersoonInOprichtingType;
+import nl.kvk.schemas.schemas.hrip.catalogus._2015._02.EenmanszaakMetMeerdereEigenarenType;
+import nl.kvk.schemas.schemas.hrip.catalogus._2015._02.BuitenlandseVennootschapType;
+
+import org.apache.commons.lang3.builder.ReflectionToStringBuilder;
+
+public class Persoon {
+    public String getId() {
+        if (bsn != null) {
+            return "nhr.bsn." + bsn;
```

Comment:
let op dat BSN niet als sleutel gebruikt mag worden want dan loop je het risico dat er een bijzonder persoonsgegeven wordt gelekt; in de huidige versie hebben we dat opgelost door de bsn te hashen en die als key te gebruiken voor natuurlijke, ingeschreven personen.
voor niet ingeschreven personen wordt daar ook een ID gemaakt.
zie https://github.com/B3Partners/brmo/blob/c586b6b47650cea3faddbde4ee5d24c0dc1b1ab0/brmo-loader/src/main/java/nl/b3p/brmo/loader/xml/NhrXMLReader.java#L133-L181